### PR TITLE
abe quickstart

### PR DESCRIPTION
**Pull request by adean20csc**
This only covers through ABE. 
Will submit another PR for other projects as add to this.

### CODE REVIEW

**Comment by jcpowermac** [comment]
Review :+1:

File: docsite/source/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+.. code-block:: bash
+
+ chmod 0600 ~/github.pem
+
+With the environment variables defined and the ssh key file created, the build script can be launched:
+​
+
+.. code-block:: bash
+
+ curl https://raw.githubusercontent.com/csc/dcaf-abe/master ansible/build.sh | bash​
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: Missing `/` between master and ansible.

File: docsite/source/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+Install the RHEL OS
+-------------------
+Install the desired version of RHEL OS. This can be on a physical or virtual machine as long as all requirements are met. Be sure to set the hostname and static ip address.
+
+For information on how to install Red Hat refer to the `Red Hat Install guide <https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/sect-installation-graphical-mode-x86.html>`_
+
+For more information on setting a static ip address refer to the `networking guide using the command line interface <https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Networking_Guide/sec-Using_the_Command_Line_Interface.html>`_.
+
+Now that the OS has been installed it is time to stage the automation resources.
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: This line can probably be dropped, we are not really staging, just scripted build.

**Comment by mtnbikenc** [comment]
Review :+1:

**Pull request merged**